src/libraries/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -485,6 +485,19 @@ public bool IsIPv6Teredo
             }
         }
 
+        /// <devdoc>
+        ///   <para>
+        ///     Determines if an address is an IPv6 Unique Local address
+        ///   </para>
+        /// </devdoc>
+        public bool IsIPv6UniqueLocal
+        {
+            get
+            {
+                return IsIPv6 && ((_numbers[0] & 0xFE00) == 0xFC00);
+            }
+        }
```

Comment:
Thanks for the PR.  To be complete, this also needs to include the new member in the System.Net.Primitives/ref/System.Net.Primitives.cs reference assembly file, and tests.